src/core/fonts.js

```diff
@@ -999,7 +1000,10 @@ class Font {
       this.fallbackName = "sans-serif";
     }
 
-    this.systemFontInfo = properties.systemFontInfo;
+    if (this.systemFontInfo?.mustGuessFallback) {
+      this.systemFontInfo.css = `${this.systemFontInfo.css},${this.fallbackName}`;
```

Comment:
This would be shorter as:
```suggestion
      this.systemFontInfo.css += `,${this.fallbackName}`;
```